src/lib/components/events/event-item.svelte

```diff
@@ -174,6 +174,7 @@
     padding: 0;
     padding: 12px 24px;
     width: auto;
+    text-align: center;
```

Comment:
Did this work as well?

Reply:
This centered the "Join" text as expected.